### PR TITLE
Update toolchain and related dependencies.

### PR DESCRIPTION
**Pull request by chinmaygarde**

### CODE REVIEW

File: tools/gn [hunk 1]
```diff
@@ -216,7 +216,7 @@ def main(argv):
   if sys.platform.startswith(('cygwin', 'win')):
     subdir = 'win'
   elif sys.platform == 'darwin':
-    subdir = 'mac'
+    subdir = 'mac-x64'
```
Comment (cbracken) [inline tools/gn]: I can’t remember if the clang-format presubmit references this but worth checking.

Comment (chinmaygarde) [reply tools/gn]: Where are the presubmits?

Comment (cbracken) [reply tools/gn]: `travis/[...]`, though looks like it should be fine.

**Comment by chinmaygarde** [comment]
Commit message has been updated and the linux builds verified.

**Comment by rmacnak-google** [comment]
LGTM

**Comment by Hixie** [comment]
@chinmaygarde What's the status of this PR?

**Comment by chinmaygarde** [comment]
I have given up on this for now. We need to contact Travis Support and figure out if we can increase the size of the disk available on the bot, or, trim dependencies in the toolchain to be smaller.

**Pull request closed without merging**